rst_files/linear_algebra.rst

```diff
@@ -1099,27 +989,18 @@ As expected, the image :math:`Av` of each :math:`v` is just a scaled version of
     :class: collapse
 
     A = [1 2
-        2 1]
+         2 1]
     evals, evecs = eigen(A)
-    a1, a2 = evals[1], evals[2]
-    evecs = evecs[:, 1], evecs[:, 2]
-    eig_1 = zeros(2, length(evecs))
-    eig_2 = zeros(2, length(evecs))
-    labels = []
-
-    for i ∈ eachindex(evecs)
-        v = evecs[i]
-        eig_1[2, i] = v[1]
-        eig_2[2, i] = v[2]
-    end
-
-    x = range(-5, stop = 5, length = 10)
+    a1, a2 = evals
+    evecs = selectdim.(Ref(evecs), 1, 1:2)
```

Comment:
Is this complexity really necessary?   I am not sure what this code is doing.  Why not just use a simple slice?

Reply:
The slice version is
```
[ evecs[i,:] for i in 1:size(evecs, 1) ]
```
If that is more clear I can do that one.

Reply:
That looks pretty clear to me, or something along those lines.